backstopper-jaxrs/build.gradle

```diff
@@ -0,0 +1,19 @@
+evaluationDependsOn(':')
+
+dependencies {
+    compile(
+            project(":backstopper-servlet-api"),
+            project(":backstopper-jackson"),
+            "javax.ws.rs:javax.ws.rs-api:2.0.1",
```

Comment:
Pull the version number out into a variable defined in the base build.gradle similar to how the other dependency versions are handled.